
passerine.exception
###################

.. note::

    This page is automatically generated. If you don't see anything, this means
    this sub-module is not meant to be used. If you really want to know what it
    is, please check out the source code at :file:`passerine/exception.py`.

.. automodule:: passerine.exception
    :members:
